packages/docusaurus-plugin-content-docs/src/client/index.ts

```diff
@@ -86,10 +86,20 @@ export const useAllDocsData = (): {[pluginId: string]: GlobalPluginData} =>
       }
     | undefined) ?? StableEmptyObject;
 
-export const useDocsData = (pluginId: string | undefined): GlobalPluginData =>
-  usePluginData('docusaurus-plugin-content-docs', pluginId, {
-    failfast: true,
-  }) as GlobalPluginData;
+export const useDocsData = (pluginId: string | undefined): GlobalPluginData => {
+  try {
+    return usePluginData('docusaurus-plugin-content-docs', pluginId, {
+      failfast: true,
+    }) as GlobalPluginData;
+  } catch (error) {
+    throw new Error(
+      `You are using a feature of the Docusaurus docs plugin, but this plugin does not seem to be enabled${
+        pluginId === 'Default' ? '' : ` (pluginId=${pluginId}`
```

Comment:
frontend-side constant available here: https://docusaurus.io/docs/docusaurus-core#constants

it's "default" lowercase BTW ;)